resources/datahub/common_scheme.go

```diff
@@ -25,6 +25,16 @@ var generalAttributes = map[string]schema.Attribute{
 			stringplanmodifier.UseStateForUnknown(),
 		},
 	},
+	"polling_options": schema.SingleNestedAttribute{
+		MarkdownDescription: "Polling related configuration options that could specify various values that will be used during CDP resource creation.",
+		Optional:            true,
+		Attributes: map[string]schema.Attribute{
+			"polling_timeout": schema.Int64Attribute{
```

Comment:
Can you give these a default value? First time user might have no idea how long to set it.